cities/moers.json

```diff
@@ -0,0 +1,93 @@
+{
+  "type": "FeatureCollection",
+  "metadata": {
+    "data_source": {
+      "title": "Offene Daten Moers, aktualisiert am 09.04.2016",
+      "url": "https://www.moers.de/de/opendataxml/maerkte/"
+    },
+    "map_initialization": {
+      "coordinates": [
+        6.642952,
+        51.455816,
```

Comment:
@odadev The `,` has to be removed. Please update the commit with `git commit --amend` and `git push --force` this branch.